tests/func/test_checkout.py

```diff
@@ -744,3 +744,26 @@ def test_checkout_dir_compat(tmp_dir, dvc):
     remove("data")
     dvc.checkout()
     assert (tmp_dir / "data").read_text() == {"foo": "foo"}
+
+
+def test_checkout_pull_option(tmp_dir, dvc, copy_script):
```

Comment:
Please add a test for `fetch`.

Reply:
I see dvc/testing/benchmarks/cli/commands/test_fetch.py. Is that the right place to add this test?

Reply:
It's https://github.com/iterative/dvc/blob/88c4b9525d83e863902790b9acdbefb82e51d83e/tests/func/test_data_cloud.py.

Reply:
Thanks!

Reply:
It might be easier to write a test for `pull`. I am okay with either of them.